pkg/services/notification/resource_control.go

```diff
@@ -107,3 +112,49 @@ func GetNotification(notificationId string) (*models.Notification, error) {
 	}
 	return nf, nil
 }
+
+func GetEmailServiceConfig(ctx context.Context) (*pb.EmailServiceConfig, error) {
+	mycfg := config.GetInstance()
+
+	protocol := mycfg.Email.Protocol
+	emailHost := mycfg.Email.EmailHost
+	port := mycfg.Email.Port
+	displayEmail := mycfg.Email.DisplayEmail
+	email := mycfg.Email.Email
+	password := mycfg.Email.Password
+	sslEnable := mycfg.Email.SSLEnable
+
+	var err error
+	if emailHost == "" {
```

Comment:
If SetServiceConfig, need to check params, if GetServiceConfig, no need to check.